[2023-08-31 13:22:17,340] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2021-10-05T00:00:00+00:00 [queued]>
[2023-08-31 13:22:17,388] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2021-10-05T00:00:00+00:00 [queued]>
[2023-08-31 13:22:17,389] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-08-31 13:22:17,390] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2023-08-31 13:22:17,390] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-08-31 13:22:17,444] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2021-10-05T00:00:00+00:00
[2023-08-31 13:22:17,451] {standard_task_runner.py:52} INFO - Started process 3024 to run task
[2023-08-31 13:22:17,460] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'second_task', '2021-10-05T00:00:00+00:00', '--job-id', '205', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmptt7bk3o9', '--error-file', '/tmp/tmpqes1pe0p']
[2023-08-31 13:22:17,464] {standard_task_runner.py:77} INFO - Job 205: Subtask second_task
[2023-08-31 13:22:17,649] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.second_task 2021-10-05T00:00:00+00:00 [running]> on host e7718740c1e6
[2023-08-31 13:22:17,846] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=devendra
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-05T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-05T00:00:00+00:00
[2023-08-31 13:22:17,847] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2023-08-31 13:22:17,850] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2023-08-31 13:22:17,880] {bash.py:169} INFO - Output:
[2023-08-31 13:22:17,933] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2023-08-31 13:22:17,941] {bash.py:177} INFO - Command exited with return code 0
[2023-08-31 13:22:18,038] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211005T000000, start_date=20230831T132217, end_date=20230831T132218
[2023-08-31 13:22:18,570] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-31 13:22:18,630] {local_task_job.py:146} INFO - Task exited with return code 0
